### PR TITLE
Improve Build and run section

### PR DESCRIPTION
**Pull request by nilminiwso2**

### CODE REVIEW

File: en/docs/build-and-run/build-and-run.md [hunk 1]
```diff
@@ -1,69 +1,78 @@
...
+# Build and run a Ballerina program
 
-Follow the steps below to create a sample Ballerina program in VS Code. 
+You can build your Ballerina program and run it either by using the **Code** view or the **Diagram** view when you use the Ballerina extension.
 
-## Use the Code View
+## Use the Code view
 
->**Note:** Skip 1-3 steps if you already have a Ballerina code.
+Let's build and run a sample Ballerina program using the Code view.
```
Comment (nilminiwso2) [inline en/docs/build-and-run/build-and-run.md:7]: Shall we move the `run a sample Ballerina program` part (steps 1 to 3) under Get Started and say `Try your sample` or `Try your first Ballerina program`?

File: en/mkdocs.yml [hunk 2]
```diff
@@ -44,8 +44,8 @@ nav:
   - Release notes:
       - Version 3.3.0: release-notes/version-3.3.0.md
   - Build and run:
-      - Run: build-and-run/build-and-run.md
-      - Tryit Tools:
+      - Build and run: build-and-run/build-and-run.md
+      - Tryit tools:
```
Comment (nilminiwso2) [inline en/mkdocs.yml:48]: Shouldn't these tools be under `Test`?

File: en/docs/build-and-run/build-and-run.md [hunk 3]
```diff
@@ -1,69 +1,78 @@
...
+
+>**Note:** If you already have a Ballerina code that you would like to build and run, you can skip steps 1 to 3 given below.
   
-1. Click **View** in the menu bar of the editor, and click **Command Palette**.
+1. Clik **View** > **Command Palette** on the VS Code editor.
 
-    >**Tip:** You can use the shortcut methods `⌘ + ↑ + P` on Mac and `Ctrl + Shift + P` on Windows and Linux.
+    >**Tip:** You can use the shortcut methods `⌘ + ↑ + P` on MacOS and `Ctrl + Shift + P` on Windows and Linux.
```
Comment (anupama-pathirage) [inline en/docs/build-and-run/build-and-run.md:13]:
```suggestion
    >**Tip:** You can use the shortcut methods `⌘ + ↑ + P` on macOS and `Ctrl + Shift + P` on Windows and Linux.
```

File: en/docs/build-and-run/build-and-run.md [hunk 4]
```diff
@@ -1,69 +1,78 @@
...
-## Use the Code View
+## Use the Code view
 
->**Note:** Skip 1-3 steps if you already have a Ballerina code.
+Let's build and run a sample Ballerina program using the Code view.
+
+>**Note:** If you already have a Ballerina code that you would like to build and run, you can skip steps 1 to 3 given below.
   
-1. Click **View** in the menu bar of the editor, and click **Command Palette**.
+1. Clik **View** > **Command Palette** on the VS Code editor.
```
Comment (anupama-pathirage) [inline en/docs/build-and-run/build-and-run.md:11]:
```suggestion
1. Click **View** > **Command Palette** on the VS Code editor.
```

File: en/docs/build-and-run/build-and-run.md [hunk 5]
```diff
@@ -1,69 +1,78 @@
...
-3. Select the **Hello World Main** example.
+3.  Click **Ballerina: Show Examples** and select the **Hello World Main** example.
 
-4. Run the program.
+4. Run the program using one of the following options:
 
     - **Option 1**
       
-        Click on the **Run** CodeLens on the editor. 
+        Click the **Run** CodeLens on the editor:
```
Comment (anupama-pathirage) [inline en/docs/build-and-run/build-and-run.md:23]: Can we combine the two lines into one? for better clarity and to reduce unnecessary space usage?

E.g.

- **Option 1** - Click the **Run** CodeLens on the editor:

**Comment by nilminiwso2** [comment]
Invalid. Replaced by new PR.

**Pull request closed without merging**